frontend/src/concepts/pipelines/content/configurePipelinesServer/ConfigurePipelinesServerModal.tsx

```diff
@@ -35,7 +33,9 @@ export const ConfigurePipelinesServerModal: React.FC<ConfigurePipelinesServerMod
   open,
 }) => {
   const { project, namespace } = usePipelinesAPI();
-  const dataConnections = useContextResourceData<DataConnection>(useDataConnections(namespace));
+  const {
+    dataConnections: { data: dataConnections },
+  } = React.useContext(ProjectDetailsContext);
```

Comment:
This breaks being able to add it with an existing connection on the global pages...